src/compiler/utilities.ts

```diff
@@ -2344,8 +2377,10 @@ namespace ts {
         return escapedCharsMap.get(c) || get16BitUnicodeEscapeSequence(c.charCodeAt(0));
     }
 
-    export function isIntrinsicJsxName(name: string) {
-        const ch = name.substr(0, 1);
+    export function isIntrinsicJsxName(name: UnderscoreEscapedString | string) {
+        // An escaped identifier had a leading underscore prior to being escaped, which would return true
+        // The escape adds an extra underscore which does not change the result
+        const ch = (name as string).substr(0, 1);

```

Comment:
It was already here? It's used within the checker and the jsx transform. Also, a JSX intrinsic is just any tag that starts with a lowercase letter (ie, `body`), indicating that its type needs to be looked up globally rather than locally, right?